### PR TITLE
Brian-Phillippi adds Ham Lake file

### LINKED ISSUE #5: Brian-Phillippi practices using GitHub Flow
- [x] Create a branch off of 'master'
- [x] add some commits
- [x] open a pull request
- [x] Collaborate :star: 
- [x] Merge when we get at least 1 :+1:

### PR DESCRIPTION
**Pull request by Brian-Phillippi**
The purpose of this pull request is to get my new code into production as quickly as possible.  If you find any mistakes be sure to ignore them.

This PR closes #5

### CODE REVIEW

File: Brian-Phillippi-HamLake.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Ham Lake, MN
+
+Population 30,000
+
+Be sure to visit Ham Lake Lanes
+
```
Comment (ericlarsen13) [inline Brian-Phillippi-HamLake.txt:6]: Don't forget about Majestic Oaks! @Brian-Phillippi

File: Brian-Phillippi-HamLake.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Ham Lake, MN
+
+Population 30,000
+
```
Comment (ericlarsen13) [inline Brian-Phillippi-HamLake.txt:4]: Don't forget about Majestic Oaks! @Brian-Phillippi

**Comment by Brian-Phillippi** [comment]
@ericlarsen13 Added Majestic Oaks to the file!

File: Brian-Phillippi-HamLake.txt (hunk 2 above)
Comment (Brian-Phillippi) [inline Brian-Phillippi-HamLake.txt]: @ericlarsen13 sorry, forgot the reference d208d31

**Comment by Brian-Phillippi** [comment]
@ericlarsen13 Forgot the reference d208d31

**Review by ericlarsen13 — APPROVED** [review]
Looks good!

**Pull request merged**